controllers/articles-controller.js

```diff
@@ -16,12 +16,12 @@ function getAllArticles(request, response, next){
     })
 }
 
-function patchVoteCount(request, response, next){
-    incrementVoteCount(request.body.inc_votes, request.params.article_id).then((updatedArticle) => {
+function patchArticleVoteCount(request, response, next){
+    incrementArticleVoteCount(request.body.inc_votes, request.params.article_id).then((updatedArticle) => {
```

Comment:
I think declaring variables and then passing these to the model would look cleaner e.g.

const {body: {inc_votes}} = request
const{params: {article_id}} = request

incrementArticleVoteCount(inc_votes, article_id)